lib/VOMSLibrary.py

```diff
@@ -43,6 +44,30 @@ def compare_dates(self, date0, date1):
 		else:
 			return 1
 
+	def parse_vomses_files(self, vomses_dirs: list = ["/etc/vomses", "~/.glite/vomses"]) -> dict:
```

Comment:
a _vomses_ file can either be a directory or a regular file, but I don't know how general you want to make this parsing

Reply:
I now check whether it's a file or a directory